src/en/contact-us.md

```diff
@@ -0,0 +1,25 @@
+Title: Contact Us
+
+# Contact us
+
+We'd love to hear about how you're using Juju, whether it's at the smallest of
+scales or the largest. Our team is always approachable and can usually be found
+in the following locations:
+
+- Our [YouTube channel][youtube] hosts the *Juju Show* and event-specific
+   broadcasts and demos, such as updates from the various Juju Charmer Summits.
+   The *Juju Show* is broadcast regularly, and features the latest developments,
+   community news, demos and guest speakers. 
+
+- Send us a message on Twitter using [`#jujucharms`][twitter].
+
+- Join us on IRC. We can be found on the [`juju`][jujuchannel] channel on [freenode][freenode].
```

Comment:
Consistency. "YouTube channel" is a visible link but for IRC only "juju" is a visible link.